documentation/accounting/MANUAL_TESTING.md

```diff
@@ -0,0 +1,165 @@
+# Madeline Accounting Features
+## Manual Testing Protocol
+
+## Setup
```

Comment:
It might be good to put the dev env setup in a different file from the manual testing protocol, because some devs (e.g. Melody) might want to setup their env but don't need to do manual testing and might not think to look in this file...